contracts/syndicate/DaiBackstopSyndicate.sol

```diff
@@ -99,28 +99,36 @@ contract DaiBackstopSyndicate is IDaiBackstopSyndicate, SimpleFlopper, ERC20 {
     _burn(msg.sender, backstopTokenAmount);
 
     // Determine the Dai currently being used to bid in auctions.
-    uint256 daiLockedInAuctions = _getActiveAuctionDaiTotal();
+    uint256 vatDaiLockedInAuctions = _getActiveAuctionVatDaiTotal();
 
     // Determine the Dai currently locked up on behalf of this contract.
-    uint256 daiBalance = _VAT.dai(address(this)) / 1e27;
+    uint256 vatDaiBalance = _VAT.dai(address(this));
 
     // Combine Dai locked in auctions with the balance on the contract.
-    uint256 combinedDai = daiLockedInAuctions.add(daiBalance);
+    uint256 combinedVatDai = vatDaiLockedInAuctions.add(vatDaiBalance);
 
     // Determine the Maker currently held by the contract.
     uint256 makerBalance = _MKR.balanceOf(address(this));
 
     // Determine the amount of Dai and MKR to redeem based on the share.
-    daiRedeemed = combinedDai.mul(shareFloat) / 1e18;
+    uint256 vatDaiRedeemed = combinedVatDai.mul(shareFloat) / 1e18;
     mkrRedeemed = makerBalance.mul(shareFloat) / 1e18;
 
+    // daiRedeemed is the e18 version of vatDaiRedeemed (e45). Needed for dai token, otherwise we keep decimals of vatDai
+    daiRedeemed = vatDaiRedeemed / 1e27;
+
+
     // Ensure that sufficient Dai liquidity is currently available to withdraw.
     require(
-      daiRedeemed <= daiBalance, "DaiBackstopSyndicate/defect: Insufficient Dai (in use in auctions)"
+      vatDaiRedeemed <= vatDaiBalance, "DaiBackstopSyndicate/defect: Insufficient Dai (in use in auctions)"
     );
 
-    // Redeem the Dai and MKR.
-    _DAI_JOIN.exit(msg.sender, daiRedeemed);
+    // Redeem the Dai and MKR, giving user vatDai if global settlement, otherwise, tokens
+    if (isEnabled()) {
```

Comment:
I think we moved to the convention of explicitly referencing calls, so `SimpleFlopper.isEnabled()` is probably preferred here!